lib/damn.js

```diff
@@ -34,24 +34,25 @@ class Damn {
   }
 
   checkAccessToken () {
-    return this.placebo().then(data => data.status === 'success')
+    return this.placebo()
+      .then(data => data.status === 'success')
   }
 
-  galleryAll (username) {
+  galleryAll (username, qs) {
     return this.client
-      .get('gallery/all', { username })
+      .get('gallery/all', Object.assign({}, { username }, qs)
```

Comment:
This makes more sense, although I don't think we need the first empty object, the following should do the same:
```js
.get('gallery/all', Object.assign({ username }, qs)
```

Reply:
You're right.